### PR TITLE
[WIP]modifying comp tool reference config

### PR DESCRIPTION
**Pull request by kbhawkey**
Updated import reference configuration.
TODO: copycomp target needs updating.
Testing output.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Looks like I put the kubeadm files in the wrong place!

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm.md [hunk 1]
```diff
@@ -1,30 +1,64 @@
...
-* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init) to bootstrap a Kubernetes control-plane node
-* [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join) to bootstrap a Kubernetes worker node and join it to the cluster
-* [kubeadm upgrade](/docs/reference/setup-tools/kubeadm/kubeadm-upgrade) to upgrade a Kubernetes cluster to a newer version
-* [kubeadm config](/docs/reference/setup-tools/kubeadm/kubeadm-config) if you initialized your cluster using kubeadm v1.7.x or lower, to configure your cluster for `kubeadm upgrade`
-* [kubeadm token](/docs/reference/setup-tools/kubeadm/kubeadm-token) to manage tokens for `kubeadm join`
-* [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset) to revert any changes made to this host by `kubeadm init` or `kubeadm join`
-* [kubeadm version](/docs/reference/setup-tools/kubeadm/kubeadm-version) to print the kubeadm version
-* [kubeadm alpha](/docs/reference/setup-tools/kubeadm/kubeadm-alpha) to preview a set of features made available for gathering feedback from the community
+
+### Synopsis
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm.md]: this seems to replace authored content with docs that are printed on the kubeadm CLI output.

Comment (neolit123) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm.md]: we didn't replace this file last cycle. perhaps the tooling changed somehow this cycle?

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm_alpha.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
+### Synopsis
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm_alpha.md]: i think the problem is that these files should be under /generated.

**Comment by kbhawkey** [comment]
I am regenerating now.

**Pull request closed without merging**